module/platform/Webots/src/Webots.cpp

```diff
@@ -562,18 +562,22 @@ namespace module::platform {
 
         if (sensor_measurements.accelerometers.size() > 0) {
```

Comment:
If this is false, doesn't that mean that the accelerometers are default values (because protobuf) and should be set as such?